front/src/statistics/Statistics.js

```diff
@@ -0,0 +1,319 @@
+                value={date}
+                // eslint-disable-next-line no-shadow
+                onChange={(date) => handleDateChange(date)}
+                onAccept={handleDatePickChange}
+                format="DD.MM.YYYY"
+                showTodayButton
+              />
+            </MuiPickersUtilsProvider>
+            <div className="continue">
+              <Button type="submit" variant="contained" style={{ backgroundColor: '#d1ccc2' }}>{statistics.DayChoose[fin]}</Button>
+            </div>
+          </form>
+        </div>
+        <hr />
+
+        {/* Header with arrows */}
+        <Grid class="date-header">
+          <div
+            className="hoverHand arrow-left"
+            onClick={previousDayClick}
+          />
+          <h1 className="headerText">
+            {date.toLocaleDateString('fi-FI')}
+          </h1>
+          <div
+            className="hoverHand arrow-right"
+            onClick={nextDayClick}
+          />
+        </Grid>
+        {/* Charts */}
+        <div className="row">
+          <div className="mixed-chart">
+            {/* Labels */}
+            <h2>{statistics.Day[fin]}</h2>
+            <h3>
+              {/* eslint-disable-next-line */}
+              {statistics.Total[fin]} {date.toLocaleDateString('fi-FI')}: {monthlyUsers[dayNumber - 1]}
+            </h3>
+            <div className="bar">
+              <Chart
+                options={dayOptions}
+                series={daySeries}
+                type="bar"
+                width="700"
+                height="400"
+              />
+            </div>
+            {/* Labels */}
+            <h2>{statistics.Month[fin]}</h2>
+            <h3>
+              {/* eslint-disable-next-line */}
+              {statistics.Total[fin]} {(date.getMonth()+1)}/{date.getFullYear()}: {total}
+            </h3>
+            <div className="line">
+              <Chart
+                options={monthOptions}
+                series={monthSeries}
+                type="line"
+                width="700"
+                height="400"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+  return (<div />);
+};
+
+// eslint-disable-next-line consistent-return
+async function getMonthlyVisitors(firstDate, lastDate) {
+  const query = `api/daterange/freeform/${firstDate}/${lastDate}`;
+  const response = await axios.get(query);
+  if (response) {
+    // Form an array including the visitors of a certain month and return it
+    let visitors = [];
+    response.data.forEach(supervision => { // eslint-disable-line
+      if (!supervision?.scheduleId) {
+        visitors = visitors.concat(0);
+      } else {
+        let trackVisitors = 0;
+        supervision.tracks.forEach (track => { // eslint-disable-line
+          trackVisitors += track.scheduled.visitors;
+        });
+        visitors = visitors.concat(trackVisitors);
+      }
+    });
+    return visitors;
+  }
+}
```

Comment:
remove eslint disable and return [] at the end of the function - We have to return something in case of an error.